cloudformation/mongo-opsmanager-server.template

```diff
@@ -284,7 +293,7 @@
             "IpProtocol": "tcp",
             "FromPort": "8080",
             "ToPort": "8081",
-            "SourceSecurityGroupId": { "Ref": "BackupLoadBalancerSecurityGroup" }
+            "SourceSecurityGroupId": { "Ref": "BackupLoadBalancerSecurityGroups" }
```

Comment:
Have you tested this with multiple items in the list? What happens when there is more than one group?